src/features/defi/providers/osmosis/components/OsmosisManager/Staking/Withdraw/components/Confirm.tsx

```diff
@@ -0,0 +1,254 @@
+import { Alert, AlertDescription, AlertIcon, Box, Stack } from '@chakra-ui/react'
```

Comment:
Also identical - will not add it for all components that are 1/1 but the gist of this is our DeFi modals already support both native Cosmos and Osmosis staking, we shouldn't duplicate them for Osmosis